app/src/main/res/values/strings.xml

```diff
@@ -534,5 +534,12 @@
     <string name="change_password_success_message">Password changed successfully!</string>
     <string name="tickets_for">Ticket(s) for %1$s</string>
     <string name="error_making_paypal_payment_message">Fail on making Paypal payment</string>
+    <string name="tick">tick</string>
+    <string name="general_settings_and_support">General Settings and Support</string>
+<<<<<<< HEAD
+    <string name="sociallink">Social Link</string>
+=======
+    <string name="socialLink">Social Link</string>
+>>>>>>> 51274de059a3850db03a67b047cfc9eb20eaf1e3
```

Comment:
nice

Reply:
I messed up before committing. I did a git pull before pushing the changes I made and this happened. I would like to close this pull request as I do not know to fix the problem that I am currently facing.